src/newcomponents/hub/elements/ChatBox.js

```diff
@@ -0,0 +1,70 @@
+import LastMessage from "@newcomponents/message/LastMessage";
+import globalConstants from "@helpers/constants";
+import { useMemo } from "react";
+
+import { ReactComponent as Group } from "@newicons/users/Group.svg";
+
+export default function ChatBox({
```

Comment:
I'm not sure I'm clear re what is a `box` in this context. Maybe `ConversationItem` ?